File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-0.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int idx = 0;
        int src = 0;
        while(src < nums.length){
            if(nums[idx]!=nums[src]){
                nums[++idx] = nums[src];
            }
            src++;
        }
        return idx+1;
    }
}

File: Data Structures & Algorithms/combination-target-sum/submission-0.py
class Solution:
    def combinationSum(self, nums: List[int], target: int) -> List[List[int]]:
        res, com = [], []

        def dfs(i,total):
            if total == target:
                res.append(com.copy())
                return

            if total >= target or i >= len(nums):
                return

            com.append(nums[i])
            dfs(i, total + nums[i])
            com.pop()
            dfs(i + 1, total)

        dfs(0,0)
        return res